sdb/commands/linux/vfs.py

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright 2020 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+import argparse
+from typing import Iterable
+
+import drgn
+import sdb
+
+
+class FGet(sdb.SingleInputCommand):
+    """
+    Return "struct file *" given the FD number for each task_struct piped as input.
```

Comment:
I think if you declare the input_type, we get some benefits like auto casting and type checking.  unless you are intentionally relying on `drgn.helpers.linux.fs.fget()` for that?

Reply:
No I wasn't. Thanks! input_type added